pkg/virt-controller/watch/drain/evacuation/evacuation_test.go

```diff
@@ -30,80 +28,56 @@ import (
 )
 
 var _ = Describe("Evacuation", func() {
-	var ctrl *gomock.Controller
-	var stop chan struct{}
 	var virtClient *kubecli.MockKubevirtClient
-	var fakeVirtClient *kubevirtfake.Clientset
-	var vmiSource *framework.FakeControllerSource
-	var vmiInformer cache.SharedIndexInformer
-	var nodeSource *framework.FakeControllerSource
-	var nodeInformer cache.SharedIndexInformer
-	var migrationInformer cache.SharedIndexInformer
-	var migrationSource *framework.FakeControllerSource
-	var podInformer cache.SharedIndexInformer
-	var podSource *framework.FakeControllerSource
 	var recorder *record.FakeRecorder
-	var mockQueue *testutils.MockWorkQueue[string]
-	var kubeClient *fake.Clientset
-	var migrationFeeder *testutils.MigrationFeeder[string]
-	var vmiFeeder *testutils.VirtualMachineFeeder[string]
 
 	var controller *EvacuationController
 
-	syncCaches := func(stop chan struct{}) {
-		go vmiInformer.Run(stop)
-		go migrationInformer.Run(stop)
-		go nodeInformer.Run(stop)
-		go podInformer.Run(stop)
-
-		Expect(cache.WaitForCacheSync(stop,
-			vmiInformer.HasSynced,
-			migrationInformer.HasSynced,
-			nodeInformer.HasSynced,
-			podInformer.HasSynced,
-		)).To(BeTrue())
-	}
-
 	addNode := func(node *k8sv1.Node) {
-		mockQueue.ExpectAdds(1)
-		nodeSource.Add(node)
-		mockQueue.Wait()
+		controller.nodeStore.Add(node)
+	}
+	enqueue := func(node *k8sv1.Node) {
+		controller.Queue.Add(node.Name)
 	}
 
 	expectMigrationCreation := func() {
-		migrationList, err := fakeVirtClient.KubevirtV1().VirtualMachineInstanceMigrations(k8sv1.NamespaceDefault).List(context.TODO(), metav1.ListOptions{})
+		migrationList, err := virtClient.VirtualMachineInstanceMigration(k8sv1.NamespaceDefault).List(context.TODO(), metav1.ListOptions{})
 		ExpectWithOffset(1, err).ToNot(HaveOccurred())
 		ExpectWithOffset(1, migrationList.Items).To(HaveLen(1))
 	}
+	updateKV := func(func(kv *v1.KubeVirt)) { panic("Implement me") }
```

Comment:
It will be overridden by the BeforeEach below, correct? Why not make it a simple var definition.

Reply:
I am not sure what you mean but in `BeforeEach` I need to capture the `store`. Here I just make sure we will not forget to implement this even if the structure of the tests changes drastically